src/App.js

```diff
@@ -0,0 +1,25 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import './App.css';
+import Booklist from './components/BookList';
+import Categories from './components/Categories';
+import Navbar from './components/Navigation';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <div className="App">
+        <Navbar />
+        <main>
+
+          <Routes>
+            <Route path="/" element={<Booklist />} />
+            <Route path="/categories" element={<Categories />} />
+          </Routes>
```

Comment:
As mentionned in the requirements, the 2 routes in your application, each should render a page component. For the first route, with the path `"/"`, it would be better if you rendered a page component which contains `BookList` which renders a list of books using the `Book` component and `Input` which is the form that you created to add a book to the list.

Reply:
Thank you for this suggestion also.